io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/internal/StreamGmlWriter.java

```diff
@@ -1637,7 +1646,30 @@ private void writeProperties(Group parent, Collection<? extends ChildDefinition<
 						if (values.length > 1) {
 							// TODO warning?!
 						}
-					}
+					} // end if for nilReason
+
+					if (child.getName().getLocalPart().toString().equals("codeSpace")
```

Comment:
I checked the definition of the schema in hale studio and the property uses an empty namespace, so could use a more accurate check by comparing the full qualified name to a `new QName("codeSpace")`